README.md

```diff
@@ -0,0 +1,209 @@
+# ENS
+
+[![Build Status](https://travis-ci.org/ensdomains/ens-contracts.svg?branch=master)](https://travis-ci.org/ensdomains/ens-contracts)
+
+For documentation of the ENS system, see [docs.ens.domains](https://docs.ens.domains/).
+
+## npm package
+
+This repo doubles as an npm package with the compiled JSON contracts
+
+```js
+import {
+  BaseRegistrar,
+  BaseRegistrarImplementation,
+  BulkRenewal,
+  ENS,
+  ENSRegistry,
+  ENSRegistryWithFallback,
+  ETHRegistrarController,
+  FIFSRegistrar,
+  LinearPremiumPriceOracle,
+  PriceOracle,
+  PublicResolver,
+  Resolver,
+  ReverseRegistrar,
+  StablePriceOracle,
+  TestRegistrar
+} from '@ensdomains/ens-contracts'
+```
+
+## Importing from solidity
+
+```
+// Registry
+import 'ensdomains-ens-contracts/contracts/registry/ENS.sol';
+import 'ensdomains-ens-contracts/contracts/registry/ENSRegistry.sol';
+import 'ensdomains-ens-contracts/contracts/registry/ENSRegistryWithFallback.sol';
+import 'ensdomains-ens-contracts/contracts/registry/ReverseRegistrar.sol';
+import 'ensdomains-ens-contracts/contracts/registry/TestRegistrar.sol';
+// EthRegistrar
+import 'ensdomains-ens-contracts/contracts/ethregistrar/BaseRegistrar.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/BaseRegistrarImplementation.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/BulkRenewal.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/BaseRegistrar.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/ETHRegistrarController.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/LinearPremiumPriceOracle.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/PriceOracle.sol';
+import 'ensdomains-ens-contracts/contracts/ethregistrar/StablePriceOracle.sol';
+// Resolvers
+import 'ensdomains-ens-contracts/contracts/resolvers/PublicResolver.sol';
+import 'ensdomains-ens-contracts/contracts/resolvers/Resolver.sol';
+```
+
+## Contract structure
```

Comment:
Probably don't want to include this in the readme. We'll have to update it every time we change anything, or it will soon become out of date.